js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,20 @@
+function abbreviateString(str) {
+  if (typeof (str) !== 'string') {
```

Comment:
The `typeof` is an operator, so the `()` are not required.

Reply:
yes thanks for that @ArfatSalman